client-app/pages/account/list-details.vue

```diff
@@ -259,16 +262,25 @@ async function addOrUpdateCartItem(item: PreparedLineItemType, quantity: number)
   }
 
   const itemInCart = cart.value?.items?.find((cartItem) => cartItem.productId === item.productId);
+  let response: ShortCartFragment | undefined;
+
+  pendingRequestCountByItemId.value[lineItem.id] = (pendingRequestCountByItemId.value[lineItem.id] || 0) + 1;
```

Comment:
pendingRequestCountByItemId -> pendingRequests
It's easier to read when the name is shorter